sql-plugin/src/main/scala/org/apache/spark/sql/rapids/stringFunctions.scala

```diff
@@ -1987,3 +1988,84 @@ case class GpuStringInstr(str: Expression, substr: Expression)
     }
   }
 }
+
+
+
+class GpuConvMeta(
+  expr: Conv,
+  conf: RapidsConf,
+  parent: Option[RapidsMeta[_,_,_]],
+  rule: DataFromReplacementRule) extends TernaryExprMeta(expr, conf, parent, rule) {
+
+  override def tagExprForGpu(): Unit = (expr.fromBaseExpr, expr.toBaseExpr) match {
+    case (Literal(fromBaseVal, IntegerType), Literal(toBaseVal, IntegerType))
```

Comment:
Could you use `extractLit` instead. It does the same kind of matching but a little differently for some corner cases.